src/main/webapp/gameHelper.js

```diff
@@ -34,18 +38,20 @@ function handleResponse(response) {
   const textPromise = response.json();
   textPromise.then(addDialogueToDom);
 }
- 
+
 function addDialogueToDom(dialogue) {
   const quoteContainer = document.getElementById('dialogue-container');
   dialogueArray = dialogue.split(";");
   dialogueRegex = 0;
+  deconstructsSentence();
   quoteContainer.innerText = dialogueArray[dialogueRegex];
 }
- 
+
 function nextLine() {
   const quoteContainer = document.getElementById('dialogue-container');
   if (dialogueRegex < dialogueArray.length-1){
     dialogueRegex ++;
+    deconstructsSentence();
```

Comment:
Really picky, but I don't know if deconstruct is the best verb? I think splitSentence is more accurate